grunttasks/yeoman.js

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
```

Comment:
Let's rename both this file and the config option `config`, `yeoman` is a product that we used as the initial scaffolding for the content server, which generated the file for us. The content server name is purely because we haven't updated it, let's do the right thing here.